runner_manager/backend/gcloud.py

```diff
@@ -0,0 +1,175 @@
+import time
+from typing import List, Literal
+
+from google.api_core.exceptions import NotFound
+from google.api_core.extended_operation import ExtendedOperation
+from google.cloud.compute import (
+    Image,
+    ImagesClient,
+    Instance,
+    InstancesClient,
+    Operation,
+    ZoneOperationsClient,
+)
+from pydantic import Field
+
+from runner_manager.backend.base import BaseBackend
+from runner_manager.models.backend import Backends, GCPConfig, GCPInstanceConfig
+from runner_manager.models.runner import Runner
+
+
+class GCPBackend(BaseBackend):
+    name: Literal[Backends.gcloud] = Field(default=Backends.gcloud)
+    config: GCPConfig
+    instance_config: GCPInstanceConfig
+
+    @property
+    def compute_client(self) -> InstancesClient:
```

Comment:
Would rather see this one listed as client for consistency with other backends
```suggestion
    def client(self) -> InstancesClient:
```

Reply:
there are other clients too. Or maybe be we don't set them at all and call them when needed

like `compute_v1.InstanceClient()`

Reply:
the compute_v1.InstanceClient is the same as InstanceClient or at least it should be, it's just versionning made inside the library in case there are introducing breaking changes. 

Setting just this one as client is enough, and the other image_client and stuff like this you can leave as is.

Reply:
ok